op-batcher/batcher/test_batch_submitter.go

```diff
@@ -64,3 +68,17 @@ func (l *TestBatchSubmitter) WaitOnJammingTx(ctx context.Context) error {
 	l.Log.Info("done waiting on jamming tx", "err", err)
 	return nil
 }
+
+// ForceOldestBlockIntoFuture sets the oldest block in the batcher's state
+// to an empty block with a number = math.MaxInt64.
+// It helps trigger a situation where there is a gap between the safe head of the sequencer
+// (which the batcher perviously started working from) and the oldest block stored by the batcher.
+func (l *TestBatchSubmitter) ForceOldestBlockIntoFuture() {
+	b := types.Block{}
+	h := types.Header{
+		Number: big.NewInt(math.MaxInt64),
+	}
+	f := b.WithSeal(&h)
+	l.lastStoredBlock = eth.ToBlockID(f)
+	l.state.blocks = []*types.Block{f}
```

Comment:
would't we need to make this access concurrency safe? you're calling it from a test where the batcher is running in the background.